### PR TITLE
Integrated File to Excel Conversion with Main App

### PR DESCRIPTION
**Pull request by SwaroopGenZ**

### CODE REVIEW

File: src/main/java/com/mfi/app/UrlConstants.java [hunk 1]
```diff
@@ -18,4 +20,12 @@ public class UrlConstants {
     public static final String DQI_REPORT_PAGE = "dqi_report";
     public static final String MESSAGE = "message";
     public static final String FILE = "file";
+
+    static final MediaType EXCEL_MEDIA_TYPE = MediaType.parseMediaType("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet");
+    public static final String DOCUMENT = "document";
+    public static final String TXT_INPUT = "textInput";
```
Comment (READIGenz) [inline src/main/java/com/mfi/app/UrlConstants.java:26]: please move these constants into properties file. Especially the Err constants,

File: src/main/java/com/mfi/app/HomeController.java [hunk 2]
```diff
@@ -102,4 +112,39 @@ public String HomeDisplay(Model model) {
...
+                return ResponseEntity.badRequest().body(NO_NAME_ERR);
+            }
+
+            return ResponseEntity
+                    .ok()
+                    .contentType(EXCEL_MEDIA_TYPE)
+                    .body(new InputStreamResource(inputStream));
+        } catch (IOException e) {
+            e.printStackTrace();
+            return ResponseEntity.status(500).body(SERVER_ERR);
```
Comment (READIGenz) [inline src/main/java/com/mfi/app/HomeController.java:146]: pls use constants for Error code 500

**Review by READIGenz — CHANGES_REQUESTED** [review]

File: src/main/java/com/mfi/app/UrlConstants.java (hunk 1 above)
Comment (SwaroopGenZ) [reply src/main/java/com/mfi/app/UrlConstants.java:26]: Done

File: src/main/java/com/mfi/app/HomeController.java (hunk 2 above)
Comment (SwaroopGenZ) [reply src/main/java/com/mfi/app/HomeController.java:146]: Done

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -16,3 +16,7 @@ config.folder.path=D:\\arthDataSoln_mfi\\config
 spring.servlet.multipart.max-file-size=1000MB
 spring.servlet.multipart.max-request-size=1000MB
 
+FILE_ERR=File is empty
```
Comment (READIGenz) [inline src/main/resources/application.properties:19]: Dear Swaroop, please use a different property file for error codes. please do not use aplication.properties.

**Review by READIGenz — APPROVED** [review]
approved with suggestions

**Pull request merged**